app/assets/stylesheets/govuk_publishing_components/components/govspeak/_button.scss

```diff
@@ -24,4 +24,33 @@
   .govuk-button--start {
     @extend .govuk-button--start;
   }
+
+  // stylelint-disable declaration-no-important
+  @include govuk-media-query($media-type: print) {
+    .gem-c-button {
+      display: inline-block;
```

Comment:
Buttons appear to already have a display of `inline-flex`, what's the benefit of changing it to `inline-block`?

Reply:
Yep this seems to be an error. I've reverted it.

Reply:
It looks like it's now set to `inline-flex` in the print styles, but that's already set by default, is that line needed?

Reply:
An oversight sorry, it's now deleted.